mods/cnc/chrome/multiplayer-createserver.yaml

```diff
@@ -170,7 +170,7 @@ Container@MULTIPLAYER_CREATESERVER_PANEL:
 							Height: 25
 							Font: Tiny
 							Align: Left
-							Text:  in the Advanced tab of the settings menu.
```

Comment:
This may be intenional.

Reply:
It is not present in the `common` chrome, just here in `cnc`, so I suspect this is a copy-paste oversight. (https://github.com/OpenRA/OpenRA/commit/d15e7f76fcd7d3089ed34f032bb09f0e9ee64e2f#diff-d8bb4c87bba2cb393823460726ed1c4f7f984f13175ea4a8e7c9aab9b8077ffbR173 is the change, so if it was intentional you might remember.) The menu still looks good to me:
![grafik](https://user-images.githubusercontent.com/7704140/148812941-0cec878c-1d35-4be7-abb4-28c865626efe.png)